cms/models.py

```diff
@@ -853,17 +853,25 @@ class CertificatePage(BootcampRunChildPage):
     template = "certificate_page.html"
     parent_page_types = ["BootcampRunPage"]
 
-    sub_title = models.CharField(
+    bootcamp_run_name = models.CharField(
         max_length=250,
         null=False,
         blank=False,
-        help_text="Specify the bootcamp certificate sub title.",
+        default="MIT Innovation and Entrepreneurship Bootcamp",
```

Comment:
I don't think we need any default value here.

Reply:
good catch, I planned to add in help_text.